libs/clients/company-registry/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export {
+  CompanyRegistryModule,
+  CompanyRegistryModuleConfig,
+} from './lib/company-registry.module'
+export * from '../gen/fetch'
```

Comment:
We should re-export some of the generated exports with better names. E.g.

```
export * from './lib/company-registry.api'
```

Where `company-registry.api.ts` looks something like this:

```
export {
  DefaultApi as CompanyRegistryApi,
  // Other exports/renames?
} from '../../gen/fetch'
```

We can also import CompanyRegistryApi from this module in `company-registry.module.ts`.

Reply:
Fixed.